tests/regression/36-apron/90-mine14-5b.c

```diff
@@ -46,6 +46,8 @@ int main(void) {
   pthread_create(&id2, NULL, t_fun2, NULL);
   pthread_mutex_lock(&mutex);
   __goblint_check(x==y);
+  __goblint_check(0 <= x);
+  __goblint_check(x <= 10);
```

Comment:
Are these present in Mine14b, example 5? If not, we should add a remark that we added them.

Reply:
`x == y` isn't in his example either. It's only described in text

> Example 6. Consider the program in Fig. 5.(b) that models an abstract producer/consumer, where X and Y denote the amount of resources. The nonrelational interference analysis is sufficient to prove that X is bounded thanks to the explicit tests on X, but it cannot find any bound on Y . Using the invariant interference abstraction parameterized with the octagon domain [17], it is possible to infer that X = Y is a lock invariant. This information automatically infers a bound on Y from the bound on X.

According to this description, the bounds are the entire goal, so we haven't even been checking the full example so far.